mmdet/models/roi_heads/bbox_heads/dii_head.py

```diff
@@ -168,6 +168,7 @@ def forward(self, roi_feat, proposal_feat):
         # Self attention
         proposal_feat = proposal_feat.permute(1, 0, 2)
         proposal_feat = self.attention_norm(self.attention(proposal_feat))
+        attn_feats = proposal_feat.permute(1, 0, 2)
 
         # instance interactive
         proposal_feat = proposal_feat.permute(1, 0,
```

Comment:
proposal_feat.permute(1, 0,2) -> attn_feats

Reply:
It seems that maybe I didn't really get what's your meaning. Can you provide more detail comment for our improvement? Respect~:)

Reply:
```python
attn_feats = proposal_feat.permute(1, 0, 2)
proposal_feat = attn_feats.reshape(-1, self.in_channels)
```

Reply:
Get it and many thanks.